docs/eventing/getting-started.md

```diff
@@ -0,0 +1,441 @@
+END
+```
+
+Here, the command creates a `Trigger` that searches for all `CloudEvents` of `source` `sendoff` and sends them to the event consumer `goodbye-display`.
+
+3. Verify that the `Triggers` are running correctly with the following command:
+
+```sh
+kubectl -n event-example get triggers
+```
+
+This command displays the **NAME**, **Broker**, **Subscriber_URI**, **AGE** and readiness of the `Triggers` in your namespace. You should see something like this:
+
+
+```sh
+NAME                 READY   REASON   BROKER    SUBSCRIBER_URI                                                                 AGE
+goodbye-display          True             default   http://goodbye-display.event-example.svc.cluster.local/   9s
+hello-display          True             default   http://hello-display.event-example.svc.cluster.local/    16s
+```
+
+Both `Triggers` should be ready and pointing to the correct `Broker`  and `Subscriber_URI`. If this is not the case, see the [Debugging Guide](./debugging/README.md).
+
+You have now created all of the subcomponents needed to recieve and manage events. In the next section, you will make the subcomponent that will be used to create your events.
+
+
+
+### Creating event producers
+
+Since this guide uses manual curl requests to send events, the final subcomponent you will need to make is an event producer called a `Pod`. The `Broker` is only exposed from within the Kubernetes cluster, so you will run the curl request from the `Pod`.
+
+1. To create the `Pod`, enter the following command:
+
+```sh
+kubectl -n event-example apply -f - << END
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: curl
+  name: curl
+spec:
+  containers:
+    # This could be any image that we can SSH into and has curl.
+  - image: radial/busyboxplus:curl
+    imagePullPolicy: IfNotPresent
+    name: curl
+    resources: {}
+    stdin: true
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    tty: true
+END
+```
+
+You will use this `Pod` to send events in the next section.
+
+## Sending CloudEvents to the `Broker`
+
+Now that the `Pod` is created, you can create a `CloudEvent` by sending an HTTP request to the `Broker`. 
+
+1. SSH into the `Pod` with the following command:
+
+```sh
+kubectl -n default attach curl -it
+```
+
+Now, you can make a HTTP request. To show the various types of events you can send, you will make three requests.
+
+1. To make the first request, enter the following in the SSH terminal:
+
+```sh
+curl -v "default-broker.event-example.svc.cluster.local" \
+  -X POST \
+  -H "Ce-Id: say-hello" \
+  -H "Ce-Specversion: 0.2" \
+  -H "Ce-Type: greeting" \
+  -H "Ce-Source: not-sendoff" \
+  -H "Content-Type: application/json" \
+  -d '{"msg":"Hello Knative!"}'
+```
+
+This creates a `CloudEvent` called `say-hello` which has the `type` `greeting`. When an event is sent to the `Broker`, the `Trigger` `hello-display` will activate and send it to the event consumer of the same name.
```

Comment:
I think I would not mention the id value here and just say, "This creates a CloudEvent that has the type `greeting`." (same with the next few examples). If you need to mention the id to reference it in later instructions, say, "This creates a CloudEvent that has the id `say-hello` and the type `greeting`."